core/src/main/java/org/apache/druid/math/expr/Parser.java

```diff
@@ -75,6 +75,9 @@ public static Expr parse(String in, ExprMacroTable macroTable)
   @VisibleForTesting
   static Expr parse(String in, ExprMacroTable macroTable, boolean withFlatten)
   {
+    if (log.isDebugEnabled()) {
```

Comment:
log.debug(..) has this check inside it , you can use string format instead of string concatenation and remove this check.

Reply:
also, not sure if you really need this log at all.

Reply:
@himanshug Thanks for your advice. I'll remove this check in the next commit. This is just for debug purpose.